components/usage-api/usage/v1/billing.proto

```diff
@@ -23,3 +26,14 @@ message UpdateInvoicesRequest {
 
 message UpdateInvoicesResponse {
 }
+
+message GetLatestInvoiceRequest {
+  string team_id = 1;
+  string user_id = 2;
+}
+
+message GetLatestInvoiceResponse {
+  string invoice_id = 1;
+  string currency = 2;
+  double amount = 3;
+}
```

Comment:
Do we also want to send the `team_id` or `user_id` back as part of the response too?

Reply:
I'd leave it out for now. The caller knows the user_id or team_id so I think for this case they can re-use that.